test_callback.c

```diff
@@ -5,6 +5,7 @@
 #include<stdio.h>
 
 /* Add fonction */ 
+int addition (int a, int b);
```

Comment:
Please change this function to "Subtraction" one

Reply:
done